docs/ref/models/constraints.txt

```diff
@@ -204,8 +204,7 @@ enforced immediately after every command.
 
 .. admonition:: MySQL, MariaDB, and SQLite.
 
-    Deferrable unique constraints are ignored on MySQL, MariaDB, and SQLite as
-    neither supports them.
+    Deferrable unique constraints are supported only on PostgreSQL.
```

Comment:
That's not true, deferrable unique constraints are also supported on Oracle. TBH, I don't think there is anything wrong in the current wording, because it clearly states that _"constraints are ignored"_ (not options).

Reply:
Okay. I agree it is clear.  I believe it's important to clearly distinguish between the omission of unique constraints and the omission of unique constraints options. 

Why don't we specify this part like below?

`include`

Unique constraints with include are ignored in all versions except PostgreSQL.

`nulls_distinct`

Unique constraints with nulls_distinct are ignored for databases besides PostgreSQL 15+.

what do you think so about this?